x-pack/plugins/observability_solution/observability/public/pages/alert_details/helpers/get_related_alerts_query.ts

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import type { Group } from '../../../../common/typings';
+
+export interface Query {
+  query: string;
+  language: string;
+}
+
+export const getRelatedAlertKuery = (tags?: string[], groups?: Group[]): string | undefined => {
+  const tagKueries: string[] =
+    tags?.map((tag) => {
+      return `tags: ${tag}`;
+    }) ?? [];
+  const groupKueries =
+    (groups &&
+      groups.map(({ field, value }) => {
+        return `(${[field]}: ${value} or kibana.alert.group.value: ${value})`;
```

Comment:
```suggestion
        return `(${[field]}: "${value}" or kibana.alert.group.value: "${value}")`;
```

Reply:
Fixed in [b657978](https://github.com/elastic/kibana/pull/193263/commits/b65797816833dd2d4c623362083b4d048419e6bb)